src/ShopApp.js

```diff
@@ -7,53 +7,64 @@ import {
   HeaderMenu,
   SearchInput,
 } from "./ShopApp.styled";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import carIcon from "./utils/assets/shopping-car.json";
 import searchIcon from "./utils/assets/search-icon.json";
 import Icon from "./utils/modules/Icon";
 import { Footer } from "./components/outside/footer/Footer";
+import { ProductList } from "./components/outside/home/products/productList/ProductList";
 
-function ShopApp() {
+const goToHome = () => {
+  window.location.href = "/";
+};
+
+export const ShopApp = () => {
   return (
-    <ShopAppWrapper>
-      <div className="Header">
-        <TopBar>
-          <span>The Best Ecommerce Store</span>
-        </TopBar>
-        <HeaderTitle>
-          <span>Muebles Troncoso</span>
-        </HeaderTitle>
-        <HeaderMenu>
-          <div className="searchSection">
-            <SearchInput placeholder="Search ...." />
-            <Icon
-              src={searchIcon.d}
-              name={"search-icon"}
-              width="32px"
-              height="32px"
-              fill="#fff"
-              viewbox={"0 0 24 24"}
-            />
-          </div>
+    <Router>
+      <ShopAppWrapper>
+        <div className="Header">
+          <TopBar>
+            <span>The Best Ecommerce Store</span>
+          </TopBar>
+          <HeaderTitle>
+            <span onClick={goToHome}>Muebles Troncoso</span>
+          </HeaderTitle>
+          <HeaderMenu>
+            <div className="searchSection">
+              <SearchInput placeholder="Search ...." />
+              <Icon
+                src={searchIcon.d}
+                name={"search-icon"}
+                width="32px"
+                height="32px"
+                fill="#fff"
+                viewbox={"0 0 24 24"}
+              />
+            </div>
 
-          <div className="items">
-            <Icon
-              src={carIcon.d}
-              viewbox={"0 0 512 512"}
-              fill="#a68a64"
-              width="40px"
-              height="40px"
-            />
-          </div>
-        </HeaderMenu>
-      </div>
-      <div className="content">
-        <Home />
-      </div>
-      <div className="footer">
-        <Footer />
-      </div>
-    </ShopAppWrapper>
+            <div className="items">
+              <Icon
+                src={carIcon.d}
+                viewbox={"0 0 512 512"}
+                fill="#a68a64"
+                width="40px"
+                height="40px"
+              />
+            </div>
+          </HeaderMenu>
+        </div>
+        <div className="content">
+          <Routes>
```

Comment:
Well I think what the deliverable wanted was to implement some logic to just hide and show the components but for me it's better that you already have the router implemented. 
If you put all this inside an AppRouter component would be better. And only reference it from here
```suggestion
          <AppRoutes/>
```